src/js/GaCesium.js

```diff
@@ -71,7 +71,10 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions,
     var fogEnabled = boolParam('fogEnabled', true);
     var fogDensity = floatParam('fogDensity', '0.0001');
     var fogSseFactor = floatParam('fogSseFactor', '25');
-    var terrainLevels = [8, 11, 14, 16, 17];
+    var terrainLevels = [8, 11, 14, 16, 18];
```

Comment:
You think 17 instead of 18 is better? Maybe take 16,17 and 18? (people are rarely that close)